### PR TITLE
improve description for reset type flags

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**

Improve description to be more specific about which option configuration.

**Did you add tests for your changes?**
Yes, updated snapshot.

**Does this PR introduce a breaking change?**
Maybe

**What needs to be documented once your changes are merged?**
Nothing

### CODE REVIEW

**Review by alexander-akait — APPROVED** [review]
Good idea!

File: test/__snapshots__/Cli.test.js.snap [hunk 1]
```diff
@@ -150,13 +150,13 @@ Object {
   "cache-immutable-paths-reset": Object {
     "configs": Array [
       Object {
-        "description": "Clear all items provided in configuration. List of paths that are managed by a package manager and contain a version or hash in its path so all files are immutable.",
+        "description": "Clear all items provided in \\"cache.immutablePaths\\" configuration. List of paths that are managed by a package manager and contain a version or hash in its path so all files are immutable.",
```
Comment (anshumanv) [inline test/__snapshots__/Cli.test.js.snap:153]: can we escape `\\` and wrap it in ` `` ` instead for clarity?

Comment (anshumanv) [reply test/__snapshots__/Cli.test.js.snap:153]: I think escaping `\\` should be enough, rest looks good

**Comment by sokra** [comment]
Thanks

**Pull request merged**